### ISSUE TITLE
@@ dropdown in dark mode unreadable black on grey

### ISSUE THREAD

**Issue by 31337-4554551n** [ylc395/joplin-plugin-note-link-system#23]
OS: Ubuntu 20.04.4 LTS x86_64
Joplin installed as app image Version 2.7.15 (prod, linux)

When using the @@ feature to link a note, the dropdown box that pops up is unreadable when using dark mode.
It's black on grey. 

Requesting a colour change on the text that comes up in the drop box in dark mode please?

**Comment by ylc395** [comment]
<img width="543" alt="image" src="https://user-images.githubusercontent.com/12264626/174971532-864b0203-e82d-4dbb-8248-0714229bd581.png">

Is the appearance in this screenshot unreadable? If it is, I will make a better dark mode appearance.

**Comment by 31337-4554551n** [comment]
![image](https://user-images.githubusercontent.com/26422914/175470385-07f9814c-e858-4ffe-9cd5-c2d0f3a3aa4a.png)
I would be happy if mine looked like yours

**Comment by ylc395** [comment]
what's your theme?

**Comment by 31337-4554551n** [comment]
Yaru-dark on ubuntu, and same issue on pop-os in their default dark theme

**Comment by mcx** [comment]
Yes, I'm having the same issue as well. It is possible to have the dropdown box follow the appearance of the current Joplin theme?

EDIT: Just tested it, it works fine on Ubuntu 22.04. The computer with the issue is running Ubuntu 18.04. Not sure what is the cause, but both computers are using a dark theme.

**Comment by executed** [comment]
Guys I think you need to do triple @@@ to open it correctly

**Comment by JasonWang2020** [comment]
这是来自QQ邮箱的自动回复邮件。您好，邮件已收到，本人将尽快处理，尽快给您回复。
